lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -42,10 +42,14 @@ def drop
 
       # Set `site.url` and `site.baseurl` if unset.
       def add_url_and_baseurl_fallbacks!
+        Jekyll::GitHubMetadata.log :debug, "Adding url and base url fallbacks..."
         site.config["url"] ||= Value.new("url", proc { |_c, r| r.url_without_path })
+          
+        Jekyll::GitHubMetadata.log :debug, "Url is set to #{site.config["url"]}"
```

Comment:
```suggestion
        Jekyll::GitHubMetadata.log :debug, "URL is set to #{site.config["url"]}"
```

Reply:
Also this line has more to do with `site.config["url"] ||...` than `return unless should_set_baseurl?` so why not swap this line with the blank line above it?